src/components/DocumentationTopic.vue

```diff
@@ -30,7 +30,7 @@
         />
       </DocumentationHero>
         <div class="container">
-          <Description :hasOverview="hasOverview">
+          <Description v-if="hasOverview">
```

Comment:
This logic is not exactly the same as before. The "No Overview" text was shown if there was no abstract OR no content in the primaryContentSections AND none of the items in the slot of the Description are rendered. 

Your new logic would outright hide all these components inside, if there is no abstract or content.

Reply:
I think I confused the primaryContentSection with Description; Updated the logic.